apps/asap-server/test/handlers/users/update-avatar.test.ts

```diff
@@ -0,0 +1,266 @@
+import nock from 'nock';
+import { APIGatewayProxyResult } from 'aws-lambda';
+import { config } from '@asap-hub/squidex';
+
+import { handler } from '../../../src/handlers/users/update-avatar';
+import { apiGatewayEvent } from '../../helpers/events';
+import { identity } from '../../helpers/squidex';
+import { body } from './update-avatar.fixtures';
+import { patchResponse } from './update.fixtures';
+import decodeToken from '../../../src/utils/validate-token';
+
+jest.mock('../../../src/utils/validate-token');
+
+describe('PATCH /users/{id}/avatar - validations', () => {
```

Comment:
In the frontend we are using post as you have required in the handler

Reply:
Yeah, this is just a copy & paste error. We don't look into the request method during tests.